packages/react-button/src/components/Button/Button.styles.ts

```diff
@@ -0,0 +1,192 @@
+import { makeClasses } from '@fluentui/react-theme-provider';
+
+export const useButtonClasses = makeClasses({
+  displayName: 'Button',
+  root: [
+    'ms-Button',
+    {
+      cursor: 'pointer',
+      alignItems: 'center',
+      borderStyle: 'solid',
+      display: 'inline-flex',
+      justifyContent: 'center',
+      outline: 'none',
+      position: 'relative',
+      userSelect: 'none',
+      boxSizing: 'border-box',
+      verticalAlign: 'middle',
+
+      background: 'var(--button-background, white)',
+      color: 'var(--button-contentColor)',
+
+      borderColor: 'var(--button-borderColor)',
+      borderRadius: 'var(--button-borderRadius)',
+      borderWidth: 'var(--button-borderWidth)',
+      boxShadow: 'var(--button-boxShadow)',
+
+      width: 'var(--button-width)',
+      height: 'var(--button-height)',
+      minHeight: 'var(--button-minHeight)',
+
+      paddingLeft: 16, // 'var(--button-padding)',
+      paddingRight: 16, // 'var(--button-padding)',
+      paddingTop: 0, // 'var(--button-padding)',
+      paddingBottom: 0, // 'var(--button-padding)',
+
+      transition: 'var(--button-transition)',
+      whiteSpace: 'var(--button-whiteSpace, nowrap)',
+
+      fontFamily: 'var(--button-fontFamily)',
+      fontSize: 'var(--button-fontSize)',
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      fontWeight: 'var(--button-fontWeight)' as any,
```

Comment:
ah yeah i am fixing this [here](https://github.com/microsoft/fluentui/pull/14696/files#diff-2cefdb4f50d42d742f727bda024ecde6). but since this PR is on hold, i should probably split it out

Reply:
Here is the fix: https://github.com/microsoft/fluentui/pull/14773

Reply:
tyty